lib/engine/config/game/g_18_newengland.rb

```diff
@@ -0,0 +1,541 @@
+# frozen_string_literal: true
+
+# File original exported from 18xx-maker/export-rb
+# https://github.com/18xx-maker/export-rb
+# rubocop:disable Lint/RedundantCopDisableDirective, Layout/LineLength, Layout/HeredocIndentation
+
+module Engine
+  module Config
+    module Game
+      module G18NewEngland
+        JSON = <<-'DATA'
+{
+  "filename": "18_newengland",
+  "modulename": "18NewEngland",
+  "currencyFormatStr": "$%d",
+  "bankCash": 12000,
+  "certLimit": {
+    "2": 25,
+    "3": 20,
+    "4": 16,
+    "5": 13
+  },
+  "startingCash": {
+    "2": 600,
+    "3": 400,
+    "4": 280,
+    "5": 280
+  },
+  "capitalization": "inceremental",
```

Comment:
spelled wrong

Reply:
fixed in new commit, is this the proper way to do it?